components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -345,6 +345,55 @@ - (void)testBackgroundColorForStateUpdatesBackgroundColorWithFallback {
   }
 }
 
+// Behavioral test to verify that MDCButton's `backgroundColor:forState:` matches the behavior of
+// UIButton's `titleColor:forState:`.  Specifically, to ensure that the special handling of
+// (UIControlStateDisabled | UIControlStateHighlighted) is identical.
+//
+// This test is valuable because clients who are familiar with the fallback behavior of
+// `titleColor:forState:` may be surprised if the MDCButton APIs don't match. For example, setting
+// the titleColor for (UIControlStateDisabled | UIControlStateHighlighted) will actually update the
+// value assigned for UIControlStateHighlighted, but ONLY if it has already been assigned. Otherwise
+// no update will take place.
+- (void)testBackgroundColorForStateBehaviorMatchesTitleColorForStateWithoutFallbackForward {
+  // Given
+  MDCButton *testButton = [[MDCButton alloc] init];
+  UIButton *uiButton = [[UIButton alloc] init];
+
+  // When
+  for (UIControlState state = 0; state < 8; ++state) {
```

Comment:
Trivial: using `UIControlState maxState = UIControlStateNormal | UIControlStateHighlighted | UIControlStateDisabled | UIControlStateSelected;` would help with readability.

Reply:
Done.

What does the "Trivial:" heading mean? That's a new one for me.

Reply:
Makes sense, thanks!